README.rst

```diff
@@ -38,12 +38,37 @@ Installation
 ============
 
 Install from pypi
+#################
 
 .. code-block:: console
 
     pip install pyobis
 
+Install from conda-forge
+########################
+Installing pyobis from the conda-forge channel can be achieved by adding conda-forge to your channels with:
+
+.. code-block:: console
+
+    conda config --add channels conda-forge
+    conda config --set channel_priority strict
+
+Once the conda-forge channel has been enabled, pyobis can be installed with conda:
+
+.. code-block:: console
+
+    conda install pyobis
+
+or with mamba:
+
+.. code-block:: console
+
+    mamba install pyobis
```

Comment:
While this is 100% correct we don't want to be the source of truth for complex install configuration and options. You do have the link for further info right below this command.

```suggestion
.. code-block:: console

    conda install pyobis --channel conda-forge
```